src/style.css

```diff
@@ -1 +1,96 @@
-/* Styles go here */
+@font-face {
+  font-family: "Roboto";
+  src: url(./fonts/Roboto-Medium.ttf);
+  font-weight: 500px;
+}
+
+@font-face {
+  font-family: "Roboto-bold";
+  src: url(./fonts/Roboto-Bold.ttf);
+  font-weight: 800;
+}
+
+.header {
```

Comment:
Header should be fullwidth
![image](https://user-images.githubusercontent.com/54936044/231000923-8ef7cf7f-85f6-43b5-84ee-e73be11faafe.png)

Reply:
* not fixed